CharlieBackend.Core/DTO/HomeworkStudent/HomeworkStudentRequestDto.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace CharlieBackend.Core.DTO.HomeworkStudent
+{
+    public class HomeworkStudentRequestDto
+    {
+        [Required]
+        public long HomeworkId { get; set; }
+       
+        [Required]
```

Comment:
Also here homework may be without any text

Reply:
@Hlib-mats  yes, because we have attachment and we can upload home tast into blockstorage . so i think we dont need write something in this field, but we need make validation that both field is not empty.